src/components/ClusterNodes/NodeActions/validationRules.ts

```diff
@@ -0,0 +1,40 @@
+import { PATTERNS } from '../../../config'
+
+export class ValidationRules {
+    taintKey = (key: string): { message: string | null; isValid: boolean } => {
+        const keyPrefixRegex = new RegExp(PATTERNS.KUBERNETES_KEY_PREFIX)
+        const keyNameRegex = new RegExp(PATTERNS.KUBERNETES_KEY_NAME)
+
+        if (!key) {
+            return { message: 'Key is required', isValid: false }
+        } else {
+            if (key.length > 253) {
+                return { message: 'Maximum 253 char is allowed', isValid: false }
```

Comment:
Shouldn't it be `Maximum 253 chars are allowed`?

Reply:
Done